src/tests/ftest/pool/evict_test.py

```diff
@@ -178,14 +181,14 @@ def test_evict(self):
         """
         pool = []
         container = []
-        non_pool_servers = []
+        #non_pool_servers = []
```

Comment:
Can this be removed if not used ?

Reply:
The part where this variable is used is commented (L195-200). It sounds like that part is commented due to a bug, so we need to keep it in case we uncomment.